static/templyte/src/app/download-template/download-template.component.css

```diff
@@ -0,0 +1,26 @@
+/* Styling copied from the create-template.component.css*/
```

Comment:
Do we need all of these? Please remove the ones that are not and move the common ones to the app level css file. Also use spacing consistent with other css files and consistent within this file itself.